src/arguments/song.ts

```diff
@@ -73,4 +83,42 @@ export default class extends Argument {
 			.filter((track) => (allowStreams ? true : track.info.isStream) && track.info.length <= maximumDuration)
 			.slice(0, remainingUserEntries);
 	}
+
+	private async loadQueue(message: KlasaMessage, url: string) {
+		try {
+			const rawData = await fetch<string[]>(url, FetchResultTypes.JSON);
+			return Promise.all(
+				rawData.map(async (track) => {
+					return {
+						track,
+						info: await this.decodeTrack(track)
+					};
+				})
+			);
+		} catch {
+			throw message.language.get('musicManagerImportQueueError');
+		}
+	}
+
+	private decodeTrack(track: string) {
```

Comment:
Move this function to MusicHandler so it's in the same place as `getSongs`. And then also rename it to `decodeSong` for consistency.